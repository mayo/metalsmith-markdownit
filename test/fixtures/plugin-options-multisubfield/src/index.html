---
title: Test
excerpt:
-
  subfield: "The excerpt has **bold** text in it!"
-
  subfield: "Another **bold** text !"
main:
  submain:
    subfield: "This is the submain.subfield Text mith **markdown**!"
---
# A Markdown Post

With some "amazing", _riveting_, **coooonnnntent**.
