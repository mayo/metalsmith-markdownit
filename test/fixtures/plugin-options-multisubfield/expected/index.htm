<h1>A Markdown Post</h1>
<p>With some &quot;amazing&quot;, <em>riveting</em>, <strong>coooonnnntent</strong>.</p>

<p>The excerpt has <strong>bold</strong> text in it!</p>
<p>Another <strong>bold</strong> text !</p>
<p>This is the submain.subfield Text mith <strong>markdown</strong>!</p>
